fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -220,6 +220,40 @@ then
 	log_Err_Abort "the --usejacobian option must be 'true' or 'false'"
 fi
 
+
+# ------------------------------------------------------------------------------
+#  Legacy Style Data Options
+# ------------------------------------------------------------------------------
+
+ReferenceReg=`opts_GetOpt1 "--refreg" $@`                            # In the cases when BOLD image is registered to a specified BOLD reference, this option 
+                                                                     # specifies whether to use 'linear' or 'nonlinear' registration to reference BOLD.
+                                                                     # Default is 'linear'.
+
+# Defaults
+ReferenceReg=${ReferenceReg:-linear}
+
+# ------------------------------------------------------------------------------
+#  Compliance check
+# ------------------------------------------------------------------------------
+
+MPPMode=`opts_GetOpt1 "--mpp-mode" $@`
+MPPMode=`opts_DefaultOpt $MPPMode "HCPStyleData"`
+Compliance="HCPStyleData"
+ComplianceMsg=""
+
+# -- BOLD nonlinear registration to reference image
+
+if [ "${ReferenceReg}" = 'nonlinear']; then
+  ComplianceMsg+=" --refreg=${ReferenceReg}"
+  Compliance="LegacyStyleData"
+fi
+
+check_mpp_compliance "${MPPMode}" "${Compliance}" "${ComplianceMsg}"
```

Comment:
FYI, since all of these separate PRs to GenericfMRIVolume add the common `check_mpp_compliance`, initialization of `Compliance`, and option parsing, I expect this will be a problem on merging, and you might need to make a new PR with all of the combined approved edits.

Reply:
I'm happy to merge all fMRIVolume related PRs into a new joint PR once all of them are approved.